developer-resources/workshops/August-2019-Bucharest-workshop/docs/README.md

```diff
@@ -0,0 +1,127 @@
+  * Open questions including 
+    * Governance Classification Status enum * still required for classification definitions 
+    * Mandy talked of using a trash zone to put entities after retention expires. 
+    * Need an event in stewardship when effectively ends or starts. 
+    * Decided not to account for different confidentiality levels on glossary term synonyms. Subject Area calls should expose APIs to be able to report on these 'messy' glossaries. 
+    * How much of area 0 should individual OMASes build out (should  security OMAS create : software server  as well as endpoints). Ideally IT infrastructure OMAS defines this * but OMASes may need to create area 0 content if they do not exist 
+    * When do we sync governance rules. Need to draw use cases to draw out the friction points when using multiple rangers. Talked of having the Asset representing Ranger itself  in a zone * to help define it's scope. 
+    * Governance types mapping. Who stitches deployed software components (deployed ranger rule) to the Governance Rule. 
+
+## 29th August 2019
+### Day 2: Architecture Discussion
+#### Morning Session: 
+* Open Metadata types SchemaType and SchemaAttribute optimisation
+  * With the optimisation of two entities it will reduce the cost of schema processing as well as the complexity for understanding it.
+  * Agreed that Mandy will model the types to fold in the schemaTypes into the schema attributes. 
+* Lineage Group Update for ING Use Case
+  * Virtual assets are unresolved entities in IGC representing unresolved tables from data stage templates. IGC stitches the templates to form design lineage. Need to get these virtual assets via the data stage proxy. Mandy suggested that we look to store these in Egeria - an asset place holder. 
+  * Decided data lineage should not show the unresolved entities (process template stored in Egeria)
+  * Once the entity is resolved then the data lineage should show it. This is likely to mean lineage server gets an event then queries Egeria to flesh out the entities and relationships. 
+  * when a Data Stage job is created, it is then defined in IGC, the data stage proxy polls for changes in the jobs, then it needs to tell Egeria to create design lineage. 
+  * We talked of who we stitch the lineage between processes. The thinking was that we would not have a process to process relationship; we should be able to infer this relationship from the output port from process 1 to a data store then to an input port to process 2. 
+  * For Data Stage proxy, it will send a process template event for job templates rather than a process event. 
+  * For alias case then there is an optional bind to assets. If the job overrides this then a new process is created.  
+* ING looking to define zones for IGC:
+  * For IGC that is one zone this could be part of the igc proxy config 
+  * For IGC with shared zones, have a new zone classification that could be set on a reference by the stewardship server. 
+
```

Comment:
In general, we decided that it would make sense to represent zones through a new classification rather than as a list property directly on an entity. This change will allow us to master the zones in a repository other than the entity's home repository -- important when the home repository may not be functionally able to manage zones.